mods/d2k/languages/rules/en.ftl

```diff
@@ -492,75 +491,75 @@ actor-missile-tank =
+    The Devastator is very resistant to bullets and high explosives, but vulnerable to missiles and high-caliber guns.
 
 actor-raider =
    .description = Improved Scout
       Strong vs Infantry, Light Vehicles
       Weak vs Tanks
    .name = Raider Trike
-   .encyclopedia = Raiders are similar to Trikes, but the Ordos have refined their fire power, speed and armor to create a powerful and maneuverable scout. With dual 20mm cannons, Raiders are most effective against infantry and lightly armored vehicles.
+   .encyclopedia = Raider Trikes, upgraded by House Ordos, have enhanced firepower, speed, and armor. Equipped with dual 20mm cannons, they are strong against infantry and lightly armored vehicles.
 
-    Raiders are vulnerable to most types of weaponry, though high-caliber guns are slightly less effective.
+    Raiders are vulnerable to most weapons, though high-caliber guns are slightly less effective against them.
 
 actor-stealth-raider =
    .description = Invisible Raider Trike
       Strong vs Infantry, Light Vehicles
       Weak vs Tanks
    .name = Stealth Raider Trike
-   .encyclopedia = A cloaked version of the raider, good for stealth attacks. Will uncloak when firing its machine guns.
+   .encyclopedia = A cloaked version of the Raider, good for stealth attacks. It uncloaks when it fires its machine guns.
 
 actor-deviator =
    .name = Deviator
    .description = Fires a warhead which changes
     the allegiance of enemy vehicles
-   .encyclopedia = The Deviator's missiles discharge a silicon cloud that interferes with vehicle controls - temporarily changing the allegiance of the targeted unit. Personnel are not seriously effected by the cloud.
+   .encyclopedia = Fires missiles that release a silicon cloud, temporarily altering the allegiance of targeted vehicles. Personnel are only slightly affected by the cloud.
 
-    The Deviator is vulnerable to most types of weapon, though high-caliber guns are slightly less effective.
+    The Deviator is vulnerable to most weapons, high-caliber guns are slightly less effective.
 
 meta-combat-tank-description = Main Battle Tank
       Strong vs Tanks
       Weak vs Infantry
 
 actor-combat-tank-a =
    .name = Atreides Combat Tank
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = It is effective against most vehicles but is less suited for lightly armored targets.
 
-    Atreides Combat Tanks are very resistant to bullet and heavy-explosives, but vulnerable to missiles and high-caliber guns.
+    It is very resistant to bullets and heavy explosives, but vulnerable to missiles and high-caliber guns.
 
 actor-combat-tank-h =
    .name = Harkonnen Combat Tank
-   .encyclopedia = The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.
+   .encyclopedia = It pis effective against most vehicles but is less suited for lightly armored targets.
```

Comment:
```suggestion
   .encyclopedia = It is effective against most vehicles but is less suited for lightly armored targets.
```

Also, this sentence is duplicated. You can use https://projectfluent.org/fluent/guide/references.html instead.

Reply:
The first commit is solely about deduplication...